### PR TITLE
docs: revised windows taskbar page

### PR DESCRIPTION
**Pull request by bandantonio**
#### Description of Change

Revised windows taskbar page

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: Revised windows taskbar page, unified structure of the sections

### CODE REVIEW

File: docs/tutorial/windows-taskbar.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-Electron also uses the app's dock icon to implement cross-platform features
+["Flash Frame" effect](#flash-frame), and cross-platform features
 like [recent documents][recent-documents] and
 [application progress][progress-bar].
 
-## JumpList
+### JumpList
 
 Windows allows apps to define a custom context menu that shows up when users
 right-click the app's icon in the task bar. That context menu is called
```
Comment (erickzhao) [inline docs/tutorial/windows-taskbar.md:16]:
```suggestion
right-click the app's icon in the taskbar. That context menu is called
```
spelling consistency

File: docs/tutorial/windows-taskbar.md [hunk 2]
```diff
@@ -58,25 +69,29 @@ app.setUserTasks([
...
+application, so the icon and program path specified for a task should exist until your application is uninstalled.
 
+[msdn-jumplist]: https://docs.microsoft.com/en-us/windows/win32/shell/taskbar-extensions#tasks
 
-## Thumbnail Toolbars
+### Thumbnail Toolbars
 
-On Windows you can add a thumbnail toolbar with specified buttons in a taskbar
-layout of an application window. It provides users a way to access to a
+In Windows, you can add a thumbnail toolbar with specified buttons to a taskbar
```
Comment (erickzhao) [inline docs/tutorial/windows-taskbar.md:90]:
```suggestion
On Windows, you can add a thumbnail toolbar with specified buttons to a taskbar
```

File: docs/tutorial/windows-taskbar.md [hunk 3]
```diff
@@ -140,42 +169,62 @@ status, as quoted from MSDN:
...
+
 win.setOverlayIcon('path/to/overlay.png', 'Description for overlay')
 ```
 
+[msdn-icon-overlay]: https://docs.microsoft.com/en-us/windows/win32/shell/taskbar-extensions#icon-overlays
+
+### Flash Frame
 
-## Flash Frame
+In Windows, you can highlight the taskbar button to get the user's attention.
```
Comment (erickzhao) [inline docs/tutorial/windows-taskbar.md:197]:
```suggestion
On Windows, you can highlight the taskbar button to get the user's attention.
```

File: docs/tutorial/windows-taskbar.md [hunk 4]
```diff
@@ -140,42 +169,62 @@ status, as quoted from MSDN:
...
 const win = new BrowserWindow()
+
 win.once('focus', () => win.flashFrame(false))
 win.flashFrame(true)
 ```
 
-Don't forget to call the `flashFrame` method with `false` to turn off the flash. In
-the above example, it is called when the window comes into focus, but you might
-use a timeout or some other event to disable it.
+> NOTE: Don't forget to call the `flashFrame` method with `false` to turn off
```
Comment (erickzhao) [inline docs/tutorial/windows-taskbar.md:223]:
```suggestion
> NOTE: Don't forget to call `win.flashFrame(false)` to turn off
```

conciseness

File: docs/tutorial/windows-taskbar.md [hunk 5]
```diff
@@ -124,11 +150,14 @@ const win = new BrowserWindow()
 win.setThumbarButtons([])
 ```
 
+[msdn-thumbnail]: https://docs.microsoft.com/en-us/windows/win32/shell/taskbar-extensions#thumbnail-toolbars
+
+### Icon Overlays in Taskbar
 
-## Icon Overlays in Taskbar
+In Windows, a taskbar button can use a small overlay to display application
```
Comment (erickzhao) [inline docs/tutorial/windows-taskbar.md:157]:
```suggestion
On Windows, a taskbar button can use a small overlay to display application
```

**Review by erickzhao — CHANGES_REQUESTED** [review]

**Review by taylorreilly1231 — APPROVED** [review]

**Review by erickzhao — APPROVED** [review]

**Review by zcbenz — APPROVED** [review]

**Comment by taylorreilly1231** [comment]
Hi, I'm not sure if this is a person or a bot. I don't know alot at all
about phone's, encryption or cyber malware, but all of this is located on
my phone by an ex who stalks me and he also uses python I don't know what
that is and can literally control my phone. I have never gave permission or
granted access to my phone. I have Verizon and my own phone company can't
detect or take it off. I'm going to attach a file and if you would please
try to drcrypt it.

**Pull request merged**